.rubocop.yml

```diff
@@ -504,3 +504,7 @@ RSpec/DescribeClass:
 # Example length is not necessarily an indicator of code quality
 RSpec/ExampleLength:
   Enabled: False
+
```

Comment:
The `.rubocop.yml` file is managed by modulesync and shouldn't be edited manually.  Could you disable the cop in the same manner as in https://github.com/voxpupuli/puppet-archive/commit/fcfc08607376b774860cc8a0d8035a3c770e336e by inserting the rubocop control comment at the first line of `spec/acceptance/rundeck_spec.rb`?